parachain/test-parachains/tests/wasm_executor/mod.rs

```diff
@@ -59,11 +59,10 @@ fn terminates_on_timeout() {
 
 #[test]
 fn parallel_execution() {
-	let execution_mode = execution_mode();
+	let isolation_strategy_1 = isolation_strategy();
+	let isolation_strategy_2 = isolation_strategy();
```

Comment:
nit: the naming here sounds to me as if they were two different strategies, whereis in fact it is a clone

Reply:
would an explicit `isolation_strategy_1.clone()` do or are you thinking about something else?

Reply:
I was thinking of `_clone` suffix for the second instead of `_1` and `_2`, but that is a tiny nit, feel free to ignore